packages/labs/motion/src/test/spring_test.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {LitElement} from 'lit';
+import {customElement} from 'lit/decorators.js';
+import {assert} from '@esm-bundle/chai';
+import {
+  SpringConfig,
+  Spring2DConfig,
+  SpringController,
+  SpringController2D,
+} from '../spring.js';
+
+suite('Spring', () => {
```

Comment:
Maybe not required, but what happens if the component is connected. The spring runs for a frame. Then the component is disconnected (calling `stop()`). Then what happens when the component is reconnected? Does the spring continue from where it left off?

Reply:
yes, it'll continue where it left off, since `start()` is called. I think this a simple choice. There may a way to advance time so the animation acts like it was running, but I don't think that's that useful.